hooks/nvidia-device-plugin/README.md

```diff
@@ -134,22 +143,18 @@ spec:
 ### Update the cluster
 
 ```bash
-kops update cluster gpu.example.com --yes
-kops rolling-update cluster gpu.example.com --yes
+kops update cluster gpu.example.k8s.local --yes
+kops rolling-update cluster gpu.example.k8s.local --yes
 ```
 
-### Deploy the Daemonset for the Nvidia DevicePlugin
+### Deploy the Daemonset for the NVIDIA DevicePlugin
 
 Only for DevicePlugin GPU Mode, load the deviceplugin daemonset for your
-specific environment.  This is not required for the Legacy Accelerators GPU
+specific environment. This is not required for the Legacy Accelerators GPU
 Mode.
 
 ```bash
-# For kubernetes 1.10
-kubectl create -f https://raw.githubusercontent.com/NVIDIA/k8s-device-plugin/v1.10/nvidia-device-plugin.yml
-
-# For kubernetes 1.11
-kubectl create -f https://raw.githubusercontent.com/NVIDIA/k8s-device-plugin/v1.11/nvidia-device-plugin.yml
+kubectl apply -f https://raw.githubusercontent.com/NVIDIA/k8s-device-plugin/master/nvidia-device-plugin.yml
```

Comment:
rather than referencing master perhaps we reference the latest tag? which seems to be 1.0.0-beta.5. Though it'd be nice if they weren't deleting tags...

https://github.com/NVIDIA/k8s-device-plugin/releases

Reply:
ok